Codecademy/client-modules
# fix(GlobalHeader): fix transparent header when refreshing scrolled SSR'd page

### Overview

seems that `react-use/useWindowScroll` is not isomorphic enough, but this is!





### PR Checklist

- [x] Related to JIRA ticket: [REACH-1048](https://codecademy.atlassian.net/browse/REACH-1048)



Testing (sort of a pain)
1. pull this down, run `yarn build-all` from the root (not 100% this is necessary)
2. `cd` to `portal-app` and run `npx npm-link-better --copy <path/to/gamut-labs>`
3. `yarn start` in `portal-app` and go to `localhost:3100/catalog` or some other page with a GlobalHeader
4. ensure the header background fades in when scrolled to the top of the page and fades out when scrolled down
5. refresh while at the top of the page and ensure the GlobalHeader renders with a background
6. scroll down and refresh the page and ensure the GlobalHeader renders without a background
